src/app/lib/ENSUtils.ts

```diff
@@ -1,16 +1,19 @@
-import { ethProvider } from "@/app/lib/provider";
 import { truncateAddress } from "@/app/lib/utils/text";
 import { isAddress } from "viem";
 import { cache } from "react";
+import { AlchemyProvider } from "ethers";
```

Comment:
is there a reason we don't just use the wagmi hooks for ens

Reply:
Good point. No idea!